awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx

```diff
@@ -206,13 +202,7 @@ class JobTemplateDetail extends Component {
                   value={
                     <ChipGroup showOverflowAfter={5}>
                       {summary_fields.credentials.map(c => (
-                        <Chip key={c.id} isReadOnly>
-                          <strong className="credential">
-                            {c.kind ? credentialType(c.kind) : i18n._(t`Cloud`)}
-                            :
-                          </strong>
-                          {` ${c.name}`}
-                        </Chip>
+                        <CredentialChip key={c.id} credential={c} isReadOnly />
```

Comment:
https://github.com/keithjgrant/awx/blob/222-job-results-details/awx/ui_next/src/components/Chip/CredentialChip.jsx#L11 is the tricky / unexpected part of showing this data (IMHO) and it looks like this new component is handling it so :+1: